app/src/main/java/eu/kanade/tachiyomi/ui/reader/viewer/ReaderTransitionView.kt

```diff
@@ -58,27 +78,46 @@ class ReaderTransitionView @JvmOverloads constructor(context: Context, attrs: At
     /**
      * Binds a next chapter transition on this view and subscribes to the load status.
      */
-    private fun bindNextChapterTransition(transition: ChapterTransition) {
+    private fun bindNextChapterTransition(
+        transition: ChapterTransition,
+        downloadManager: DownloadManager,
+        manga: Manga
```

Comment:
```suggestion
        manga: Manga,
```

Reply:
Out of curiosity, why add a comma here ? Just a personal preference ?